### DISCUSSION TITLE
Heightmap Import - what's expected?

### DISCUSSION THREAD

**Discussion by jjoshpoland** [TokisanGames/Terrain3D#203] [Questions] [answered]
I've created a heightmap in Krita I was using in Unity to import as a terrain heightmap, was hoping to use it here. Its a black and white 1024x1024 image where black is zero height and white is max height.

Importing it using the importer script doesn't appear to do anything. Console shows that the file is being imported and then nothing happens.

Is black and white the wrong channel? What is the tool expecting for a heightmap file?

**Comment by TokisanGames** [comment, marked as answer]
I answered you on discord but for posterity, please read the documentation I wrote. Your heightmap is probably normalized and height needs to be scaled on import.

https://github.com/TokisanGames/Terrain3D/wiki/Importing-&-Exporting-Data

You can also export the demo data and compare with the file you made. 

Also there are no black and white channels. There are red, green, blue, and alpha channels. You can save the data in a single channel grey scale image (red channel), or as rgb, doesn't matter.

Please close this if this and the documentation answers your questions.

**Comment by jjoshpoland** [comment]
Thanks for the response - yes scaling up between 300 and 500 did the trick.

**Reply by TokisanGames** [reply]
Great, thanks for the report.
